docs/admin_guides/docker_admin.md

```diff
@@ -16,24 +16,6 @@ docker-compose ps
 
 This should give you the name, entry point, current status and internal ports used within the docker network as output.
 
-## Admin Login
```

Comment:
Shouldnt we keep the admin login? Or how do we get the first administrator user?

Reply:
It is not removed, just refactor https://github.com/warlof/eveseat-docs/blob/4.0.x/docs/admin_guides/admin_login.md

In order to avoid split stuff related to same things everywhere in the documentation.